server/lib/config/pbench-server-default.cfg

```diff
@@ -126,6 +126,12 @@ lowerbound = 820
 # index_prefix =
 # bulk_action_count =
 
+[elasticsearch]
+server = elasticsearch.example.com
```

Comment:
For Elasticsearch we really need `host` and `port`.  Can we remove the `server`, `host`, and `port` options from the `Indexing` section, and move them to the `elasticsearch` section as `host` and `port` only.  Then fix up the `indexer.py` to reference both sections?

Reply:
So you mean something like this 
``` 
[Indexing]
index_prefix =
bulk_action_count =

[elasticsearch]
host = 
port = 
```
and then referencing these 2 endpoints in indexer.py's `_get_es_hosts` to get the ES host?

Reply:
It actually make sense if we have 2 separate sections in our server config file `[Indexing]` and `[elasticsearch]` where elasticsearch section holds host and port specific information and `Indexing` section holds index specific information such as index_prefix and all. However, it means we might have to update the pbench-server.cfg file